okhttp/src/androidMain/kotlin/okhttp3/OkHttp.android.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2025 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import android.content.Context
+import okhttp3.internal.CONST_VERSION
+import okhttp3.internal.platform.Platform
+import okhttp3.internal.platform.PlatformRegistry
+
+actual object OkHttp {
+  @JvmStatic
+  actual val Version: String = CONST_VERSION
+
+  @Suppress("MayBeConstant") // Non-const so external callers get the runtime version.
+  @JvmField
+  @Deprecated(level = DeprecationLevel.WARNING, message = "Replaced with Version", replaceWith = ReplaceWith("Version"))
```

Comment:
Why deprecate this?

Reply:
I guess no need. But it's not available via common now.